src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/SymbolTable.cs

```diff
@@ -819,7 +819,7 @@ private CType ProcessSpecialTypeInChain(NamespaceOrAggregateSymbol parent, Type
             else if (t.IsArray)
             {
                 // Now we return an array of nesting level corresponding to the rank.
-                ctype = _typeManager.GetArray(GetCTypeFromType(t.GetElementType()), t.GetArrayRank());
+                ctype = _typeManager.GetArray(GetCTypeFromType(t.GetElementType()), t.GetArrayRank(), t.IsSZArray);
```

Comment:
Is this use of `IsSZArray` going to be safe when building for netstandard, or do I need to conditionally bring in a kludge for that case?